pom.xml

```diff
@@ -407,6 +408,17 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.sonatype.plugins</groupId>
+				<artifactId>nexus-staging-maven-plugin</artifactId>
```

Comment:
I think this was already added to the pom under the `release` profile lower in this file; was that not having the correct effect?

Reply:
You're right! the profile was not being activated in `drop.sh` - updated PR to active the profile instead. PTAL